src/sidebar/panel.js

```diff
@@ -333,3 +337,17 @@ document.addEventListener('DOMContentLoaded', getLastSyncedTime);
 // Create a connection with the background script to handle open and
 // close events.
 browser.runtime.connect();
+
+function insertSelectedText(editor, text) {
+  const currentNoteContents = editor.getData();
+  let updatedNotesContents = '';
+  if (currentNoteContents === '<p>&nbsp;</p>') {
```

Comment:
What about if `currentNoteContents` contains the initial content ? My guess is right now it will add `text` at the end, while we probably want to remove it first.

Reply:
@ryanfeeley do we want to remove the initial context first? (if it is a fresh pad)